front-end/src/styles.scss

```diff
@@ -517,3 +517,28 @@ table {
   font-size: 1.5rem;
   margin-left: 0.5rem;
 }
+
+select {
+  max-height: 47px;
+  min-height: 47px;
+  height: 47px;
+  font-family: var(--karla);
+  font-size: 1rem;
+  font-weight: 400;
+  -webkit-appearance: none; /* Chrome, Safari, Opera */
+  -moz-appearance: none; /* Firefox */
+  appearance: none;
+  background: none;
+  background-color: transparent;
+  background-image: url("data:image/svg+xml,%3Csvg%20xmlns='http://www.w3.org/2000/svg'%20viewBox='0%200%2014%2014'%20fill='%23a1a1aa'%20aria-hidden='true'%3E%3Cpath%20d='M7.01744%2010.398C6.91269%2010.3985%206.8089%2010.378%206.71215%2010.3379C6.61541%2010.2977%206.52766%2010.2386%206.45405%2010.1641L1.13907%204.84913C1.03306%204.69404%200.985221%204.5065%201.00399%204.31958C1.02276%204.13266%201.10693%203.95838%201.24166%203.82747C1.37639%203.69655%201.55301%203.61742%201.74039%203.60402C1.92777%203.59062%202.11386%203.64382%202.26584%203.75424L7.01744%208.47394L11.769%203.75424C11.9189%203.65709%2012.097%203.61306%2012.2748%203.62921C12.4527%203.64535%2012.6199%203.72073%2012.7498%203.84328C12.8797%203.96582%2012.9647%204.12842%2012.9912%204.30502C13.0177%204.48162%2012.9841%204.662%2012.8958%204.81724L7.58083%2010.1322C7.50996%2010.2125%207.42344%2010.2775%207.32656%2010.3232C7.22968%2010.3689%207.12449%2010.3944%207.01744%2010.398Z'/%3E%3C/svg%3E");
```

Comment:
Unfortunately the use of the `data:` scheme is a csp violation with no good policy workaround.  I ran into this during our csp work and found [this discussion](https://security.stackexchange.com/a/95011/208408) helpful to understand.  

Basically, the solution is to generate the svg as an asset and link to it like [here](https://github.com/fecgov/fecfile-web-app/blob/d0578659c0d70f14ccc5143cc6201452603382d6/front-end/src/app/layout/banner/banner.component.scss#L99)